service/src/utils/arweave.ts

```diff
@@ -0,0 +1,35 @@
+import Arweave from 'arweave';
+import { decrypt } from 'ecies-geth';
+import { ethers } from 'ethers';
+import { solidityKeccak256 } from 'ethers/lib/utils';
+import { archLogger } from './chalk-theme';
+
+const arweave = Arweave.init({
+    host: 'localhost', // 'arweave.net', // Hostname or IP address for a Arweave host
+    port: 1984, //443, // Port
+    protocol: 'http', // 'https', // Network protocol http or https
+    timeout: 20000, // Network request timeouts in milliseconds
+    logging: false, // Enable network request logging
+});
```

Comment:
This config object could be dynamic -- either based on NODE_ENV (development/production) or alternatively could use an additional arweave env file with its own ENV vars (ARWEAVE_HOST, ARWEAVE_PROTOCOL, etc) , so we can easily switch between localhost chain and mainnet chain (arweave.net).